static/panes/compiler.ts

```diff
@@ -96,7 +96,7 @@ const OpcodeCache = new LRUCache<string, CachedOpcode>({
     },
 });
 
-function patchOldFilters(filters) {
+function patchOldFilters(filters: Partial<Record<string, boolean>> | undefined): Record<string, boolean> | undefined {
```

Comment:
Why is this Partial? Partial of string doesn't really make sense I don't think?

Reply:
This `Partial` means only part of the keys in the `Record` are required, doesn't it?

Reply:
Sorry yes you are right